### PR TITLE
Extract logo into new component

### PR DESCRIPTION
**Pull request by tcollier**
In playing around with the layout, I found myself moving the logo from component to component. When it is stored as a constant in the component it is rendered in, it becomes arduous to move it around. As a component, it is much simpler to move around. Also, it is now reusable in multiple components, though I doubt that will come in handy any time soon.

### CODE REVIEW

File: src/components/Logo.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            fill="#210d36"
```
Comment (lynnpark) [inline src/components/Logo.js]: Suggestion: allow the fill values to be passed int through props, with the current values as default

**Comment by lynnpark** [comment]
I think Logo could live in its own folder `/components/svgs`, since I have a feeling we'll be adding many more svgs to this project 😉

**Comment by lynnpark** [comment]
For future state: I want to create a simple wrapper component for SVGs that encapsulates logic for passing in styles. Some sort of a simple [Higher-Order Component](https://reactjs.org/docs/higher-order-components.html) maybe??

File: src/components/Logo.js (hunk 1 above)
Comment (tcollier) [reply src/components/Logo.js]: Perfect for skinning the site for holidays!

Screen Shot 2020-01-23 at 5 11 56 PM

**Pull request merged**